beets/importer.py

```diff
@@ -620,8 +621,12 @@ def align_album_level_fields(self):
         """Make some album fields equal across `self.items`.
         """
         changes = {}
+        # Determine where to gather the info from for the RETAG action.
+        retag_asis = (self.choice_flag == action.RETAG and
+                      not self.items[0].artist and
+                      not self.items[0].mb_artistid)
```

Comment:
Ah, thanks for the explanation---I understand now.

Here's a crazy proposal: what if we never apply the "as-is fixup" logic for the RETAG case? The philosophy would be: if you're willing to edit the data manually, then you can take matters into your own hands.